packages/colorpickers/src/styled/ColorPicker/StyledInput.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { Input } from '@zendeskgarden/react-forms';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'colorpickers.colorpicker_input';
+
+export const StyledInput = styled(Input as any).attrs({
```

Comment:
why not `as 'input'`?

Reply:
I think it's because `input` does not have attributes such as `isCompact`, `isBare`, etc. 

`Input as React.ForwardRefExoticComponent<IInputProps>` compiles locally, but in CI runs into the issue regarding multiple versions of `@types/react`.

Reply:
@austingreendev thots on an improved TS construct or running `npx yarn-deduplicate`? Seems like a lockfile might be less than pristine.

Reply:
`yarn-deduplicate` wasn't able to help us here as we aren't hoisting dependencies. There are 2 different versions of `@types/react` and even if they're the same version I think we would have these inference issues.

@hzhu did you try removing and re-adding `@types/react-transition-group` in `react-modals`?

Reply:
Yes, I've tried removing and re-adding `@types/react-transition-group` in `react-modals`. It'd be great to take another look at this one more time, but if it requires more dedicated time I think this issue could be addressed outside the context of this PR.